pkg/monitor/datapath_policy.go

```diff
@@ -39,6 +39,10 @@ const (
 	// corresponds to the policy match type
 	PolicyVerdictNotifyFlagMatchType = 0x38
 
+	// PolicyVerdictNotifyFlagIsAudited is the bit mask in Flags that
+	// corresponds to wether the traffic was allowed due to the audit mode
```

Comment:
```suggestion
	// corresponds to whether the traffic was allowed due to the audit mode
```

Reply:
Fixed and updated to use reference. Thanks for the suggestion.